frontends/web/src/routes/account/send/send.tsx

```diff
@@ -358,27 +359,6 @@ class Send extends Component<Props, State> {
     }
   };
 
-  private handleFormChange = (event: React.SyntheticEvent) => {
-    const target = (event.target as HTMLInputElement);
-    let value: string | boolean = target.value;
-    if (target.type === 'checkbox') {
-      value = target.checked;
-    }
-    if (target.id === 'sendAll') {
-      if (!value) {
-        this.convertToFiat(this.state.amount);
-      }
-    } else if (target.id === 'amount') {
-      this.convertToFiat(value);
-    }
-    this.setState(prevState => ({
-      ...prevState,
-      [target.id]: value,
-    }), () => {
-      this.validateAndDisplayFee(true);
-    });
-  };
-
```

Comment:
@thisconnect In regards to your comment, the whole function can actually be removed totally. This was previously used in `sendToSelf`, but it serves no purpose as the `value` when it gets called from `sendToSelf` will be undefined (it's a label).